query/src/configs/config_query.rs

```diff
@@ -181,6 +181,18 @@ pub struct QueryConfig {
     #[serde(default)]
     pub rpc_tls_query_service_domain_name: String,
 
+    #[structopt(long, env, help = "Table engine csv enabled")]
+    #[serde(default)]
+    pub table_engine_csv_enabled: bool,
+
+    #[structopt(long, env, help = "Table engine parquet enabled")]
+    #[serde(default)]
+    pub table_engine_parquet_enabled: bool,
+
+    #[structopt(long, env, help = "Table engine memory enabled")]
+    #[serde(default)]
+    pub table_engine_memory_enabled: bool,
```

Comment:
Good job, LGTM.
It seems the memory engine can be enabled by default, so we can do some memory related tests only start the databend-query and no need to enable it in the toml.

Reply:
It can't set field default true in structopt. If the field not in toml file, it thinks it is not present and make value false.